openbas-api/src/main/java/io/openbas/xtmhub/XtmHubService.java

```diff
@@ -0,0 +1,41 @@
+package io.openbas.xtmhub;
+
+import io.openbas.database.model.User;
+import io.openbas.rest.settings.response.PlatformSettings;
+import io.openbas.service.PlatformSettingsService;
+import io.openbas.service.UserService;
+import jakarta.validation.constraints.NotBlank;
+import java.time.LocalDateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Use @RequiredArgsConstructor and remove setPlatformSettingsService and setUserService, and set private PlatformSettingsService platformSettingsService;
  private UserService userService; to private final